### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by ImJC1C**
updated 
-gitignore
-imgui style

### CODE REVIEW

File: src/vulkan/ImGui.cxx [hunk 1]
```diff
@@ -354,12 +354,92 @@ void ImGui::init(task::SynchronousWindow* owning_window, vk::SampleCountFlagBits
 
   // Set up ImGui style to use.
   StyleColorsDark();
-  ImGuiStyle& gui_style = GetStyle();
-  gui_style.Colors[ImGuiCol_TitleBg] = ImVec4( 0.16f, 0.29f, 0.48f, 0.9f );
-  gui_style.Colors[ImGuiCol_TitleBgActive] = ImVec4( 0.16f, 0.29f, 0.48f, 0.9f );
-  gui_style.Colors[ImGuiCol_WindowBg] = ImVec4( 0.06f, 0.07f, 0.08f, 0.8f );
-  gui_style.Colors[ImGuiCol_PlotHistogram] = ImVec4( 0.20f, 0.40f, 0.60f, 1.0f );
-  gui_style.Colors[ImGuiCol_PlotHistogramHovered] = ImVec4( 0.20f, 0.45f, 0.90f, 1.0f );
+  ImGuiStyle& style = ImGui::GetStyle();
```
Comment (ImJC1C) [inline src/vulkan/ImGui.cxx:357]: changed name of imgui style

File: src/vulkan/ImGui.cxx [hunk 2]
```diff
@@ -354,12 +354,92 @@ void ImGui::init(task::SynchronousWindow* owning_window, vk::SampleCountFlagBits
...
+  style.ScrollbarSize = 14.0;
+  style.ScrollbarRounding = 0.0;
+  style.GrabMinSize = 10.0;
+  style.GrabRounding = 0.0;
+  style.TabRounding = 0.0;
+  style.TabBorderSize = 0.0;
+  style.TabMinWidthForCloseButton = 0.0;
+  style.ColorButtonPosition = ImGuiDir_Right;
+  style.ButtonTextAlign = ImVec2(0.5, 0.5);
+  style.SelectableTextAlign = ImVec2(0.0, 0.0);
```
Comment (ImJC1C) [inline src/vulkan/ImGui.cxx:388]: Sizing Styles

File: src/vulkan/ImGui.cxx [hunk 3]
```diff
@@ -354,12 +354,92 @@ void ImGui::init(task::SynchronousWindow* owning_window, vk::SampleCountFlagBits
...
+  style.Colors[ImGuiCol_TableBorderStrong] = ImVec4(0.3098039329051971, 0.3098039329051971, 0.3490196168422699, 1.0);
+  style.Colors[ImGuiCol_TableBorderLight] = ImVec4(0.2274509817361832, 0.2274509817361832, 0.2470588237047195, 1.0);
+  style.Colors[ImGuiCol_TableRowBg] = ImVec4(0.0, 0.0, 0.0, 0.0);
+  style.Colors[ImGuiCol_TableRowBgAlt] = ImVec4(1.0, 1.0, 1.0, 0.05999999865889549);
+  style.Colors[ImGuiCol_TextSelectedBg] = ImVec4(0.0, 0.4666666686534882, 0.7843137383460999, 1.0);
+  style.Colors[ImGuiCol_DragDropTarget] = ImVec4(0.1450980454683304, 0.1450980454683304, 0.1490196138620377, 1.0);
+  style.Colors[ImGuiCol_NavHighlight] = ImVec4(0.1450980454683304, 0.1450980454683304, 0.1490196138620377, 1.0);
+  style.Colors[ImGuiCol_NavWindowingHighlight] = ImVec4(1.0, 1.0, 1.0, 0.699999988079071);
+  style.Colors[ImGuiCol_NavWindowingDimBg] = ImVec4(0.800000011920929, 0.800000011920929, 0.800000011920929, 0.2000000029802322);
+  style.Colors[ImGuiCol_ModalWindowDimBg] = ImVec4(0.1450980454683304, 0.1450980454683304, 0.1490196138620377, 1.0);
```
Comment (ImJC1C) [inline src/vulkan/ImGui.cxx:442]: Color Styles

**Review by ImJC1C — COMMENTED** [review]
changed ImGuiStyle var name from gui_style to style
added color and sizing styles

**Comment by CarloWood** [comment]
Sorry for the extremely late reaction. I have mainly postponed looking into this because of the addition of the Visual Studio support to .gitignore. Since this project is linux-only I don't want to add any trace refering to vs; but since that was part of the pull request it made it hard to quickly test it out. I am happy with how imgui looks now anyways (it is just for debugging). I just took the time to manually apply the change and did not think it was a clear improvement either, so I am closing this as "won't fix" or whatever applies.

**Pull request closed without merging**